server/policy_constants.go

```diff
@@ -161,7 +161,7 @@ func ParseGCENonHostInfo(nonHostInfo []byte) (pb.GCEConfidentialTechnology, erro
 		return pb.GCEConfidentialTechnology_NONE, errors.New("prefix for GCE Non-Host info is missing")
 	}
 	tech := nonHostInfo[prefixLen]
-	if tech > byte(pb.GCEConfidentialTechnology_AMD_SEV_ES) {
+	if tech > byte(pb.GCEConfidentialTechnology_AMD_SEV_SNP) || tech == byte(3) {
```

Comment:
have we already reserved 3 in the proto for another technology? If so, consider commenting that it's reserved in the `GCEConfidentialTechnology` enum in attest.proto

Reply:
3 is reserved internally and will be used eventually. Will add a comment.